### PR TITLE
fetching fixes

### PR DESCRIPTION
**Pull request by dorsadeh**
1.ccc list added to default settings
2.tickers.json removed from ignore list
3.handling failed http requests

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -15,12 +15,25 @@
 def get_data(metrics: list, tickers: list):
     fetcher = data_fetcher.DataFetcher("downloaded_data")
     fetcher.init_downloads_directory()
+    failed_list = []
+    cnt = 0
```
Comment (talobs) [inline main.py:19]: The pythonic way to use counter in a for loop is to use enumerate()

File: main.py [hunk 2]
```diff
@@ -15,12 +15,25 @@
 def get_data(metrics: list, tickers: list):
     fetcher = data_fetcher.DataFetcher("downloaded_data")
     fetcher.init_downloads_directory()
```
Comment (talobs) [inline main.py:17]: The format which the pd.to_csv() function saves date is not compatible with numpy. Therefore it is best to be saved as %Y-%m-%d.

Comment (talobs) [reply main.py:17]: Done by me :)

File: main.py (hunk 1 above)
Comment (dorsadeh) [reply main.py]: nice one! I will use it from now on

**Review by talobs — APPROVED** [review]

**Pull request merged**